charts/llm-d-modelservice/templates/decode-lws.yaml

```diff
@@ -6,17 +6,23 @@ metadata:
   labels:
     {{- include "llm-d-modelservice.labels" . | nindent 4 }}
     {{- include "llm-d-modelservice.decodelabels" . | nindent 4 }}
+  annotations:
+    leaderworkerset.sigs.k8s.io/subgroup-exclusive-topology: kubernetes.io/hostname
 spec:
-  {{- if not .Values.decode.autoscaling.enabled }}
+  {{- if not (and .Values.decode.autoscaling .Values.decode.autoscaling.enabled) }}
   replicas: {{ default 1 .Values.decode.replicas }}
   {{- end }}
   leaderWorkerTemplate:
-    size: {{ div (int (include "llm-d-modelservice.dataParallelism" .Values.decode.parallelism)) (int (include "llm-d-modelservice.dataLocalParallelism" .Values.decode.parallelism)) }}
-    leaderTemplate:
+    size: {{ int (include "llm-d-modelservice.dataParallelism" .Values.decode.parallelism) }}
+    subGroupPolicy:
+      subGroupSize: 8
+    workerTemplate:
       metadata:
         labels:
           {{- include "llm-d-modelservice.decodelabels" . | nindent 10 }}
       spec:
+        hostIPC: true
+        hostPID: true
```

Comment:
Do these need to be settable from values?

Reply:
[11:35] Michael Kalantar
 In this example, I see, in workerTemplate.spec:
hostIPC: true
hostPID: true
Are these values that should always be set, might be optionally set, or were set for development?  I don’t recall from the conversation yesterday.
11:36 Robert Shaw
 These are needed at current
11:37. Tyler Michael Smith
 yeah we need these -- not optional and not development